cores/arduino/stm32/usb/cdc/usbd_cdc.c

```diff
@@ -882,6 +882,23 @@ uint8_t  USBD_CDC_ReceivePacket(USBD_HandleTypeDef *pdev)
   }
 }
 
+uint8_t USBD_CDC_ClearBuffer(USBD_HandleTypeDef *pdev)
+{
+  /* Suspend or Resume USB Out process */
+  if (pdev->pClassData != NULL) {
+    if (pdev->dev_speed == USBD_SPEED_HIGH) {
+      /* Prepare Out endpoint to receive next packet */
+      USBD_LL_PrepareReceive(pdev, CDC_OUT_EP, 0, 0);

```

Comment:
Do the same in any case, maybe the test could be removed?

Reply:
Yes! Already done